site/documentation/content/concepts/tenancy.md

```diff
@@ -72,9 +72,7 @@ is addressing the tenants themselves.
 All devices of Hono belong to exactly one tenant, so any data is sent in the context of the
 tenant a device belongs to.
 
-It is important here to focus on Hono's view of a device: it is on a logical level, meaning a device is created in the 
-context of a tenant purely by using Hono's Credentials and Device Registration API. 
-
+"Device" means here a logical entity consisting of a set of registry data.
```

Comment:
IMHO we can completely omit this sentence and rephrase the following one as
> A physical device will usually be represented in Hono as an entity in the device registry, having a unique identity and belonging to exactly one tenant.

WDYT?

Reply:
That sounds great, but there is an overlap with the previous sentence:  

> All devices of Hono belong to exactly one tenant, so any data is sent in the context of the
tenant a device belongs to. A physical device will usually be represented in Hono as an entity in the device registry, having a unique identity and belonging to exactly one tenant.

We could skip the first sentence and put it like this:

> A physical device will usually be represented in Hono as an entity in the device registry, having a unique identity and belonging to exactly one tenant. All data sent from a device, as well as from the application to the device, is therefore treated as belonging to the corresponding tenant.

WDYT?